README.md

```diff
@@ -166,15 +166,15 @@ Please visit [Zivid Knowledge Base][zivid-knowledge-base-url] for general inform
 
 ## Test matrix
 
-| Operating System | Python version            |
-| :--------------- | :------------------------ |
-| Ubuntu 23.10     | 3.11                      |
-| Ubuntu 23.04     | 3.11                      |
-| Ubuntu 22.04     | 3.10                      |
-| Ubuntu 20.04     | 3.8                       |
-| Fedora 38        | 3.11                      |
-| Fedora 37        | 3.11                      |
-| Windows 10       | 3.7, 3.8, 3.9, 3.10, 3.11 |
+| Operating System | Python version             |
+| :--------------- |:---------------------------|
```

Comment:
Why? This makes the two collums different.

Reply:
Hmm, looks like my IDE reformatted this. Not sure why it made them different.

Reply:
Reformatted the other column also.